### PR TITLE
Rendering for tailor survey

### PR DESCRIPTION
**Pull request by MahanaC**
Some more changes to tailor's survey:
- CSS modifications to make sure the styling looks right, e.g. on [https://www.theguardian.com/business/2017/mar/08/nike-launches-hijab-for-female-muslim-athletes](url)
screen shot 2017-03-09 at 13 14 53
now becomes
screen shot 2017-03-09 at 14 57 48

- Change to use space filler to find location for survey, as the current method sometimes doesn't look right (I've played around with the rules and these seem ok, but if anyone has further suggestions that would be great), e.g. on [https://www.theguardian.com/business/2017/mar/08/nike-launches-hijab-for-female-muslim-athletes](url)
screen shot 2017-03-09 at 13 10 40
now becomes 
screen shot 2017-03-09 at 13 12 55

- Tracking changes to follow @JPodmore's new naming convention

@lindseydew @oilnam @GHaberis @stephanfowler @SiAdcock @mike-ruane

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/tailor-survey.js
Comment (sndrs) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/tailor-survey.js]: does this need to be a function? could just be a variable?

**Comment by MahanaC** [comment]
Thanks @sndrs have made that change - does it look ok to merge now? If so does someone mind merging it, as I don't have write access? Thanks

**Review by SiAdcock — APPROVED** [review]

**Pull request merged**
